scm/const.go

```diff
@@ -67,6 +67,8 @@ func (a Action) String() (s string) {
 		return "synchronized"
 	case ActionMerge:
 		return "merged"
+	case ActionEdit:
```

Comment:
is this not really as update and edit are synonyms
```
case ActionUpdate:
		return "updated"
```

Reply:
@tphoney We can't keep this as Update since in triggers we have already defined an edit action on UI. We just checked since ActionEdit was not being passed Github IssueComment trigger with Edit action was not triggering so we need to pass this value. You can check below snippet.

https://github.com/drone/go-scm/blob/bff969507a83c53a7928825cc7c1c57be3c01b87/scm/driver/github/webhook.go#L135

Reply:
The purpose of this library is to provide an abstraction layer across all scm providers, so that we can use the same data structures and terminology for all providers.  This library is not intended to align with the Harness user interface, which may choose to expose scm-specific data.  This library should only be used where abstract is needed;   if you want the original data or do not need abstractions, you would not use this library.  Although perhaps I am misunderstanding.